dtschema/schemas/thermal/thermal.yaml

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: BSD-2-Clause
+# Copyright 2023 Collabora Ltd.
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/thermal/thermal.yaml#
+$schema: http://devicetree.org/meta-schemas/base.yaml#
+
+title: Thermal Consumer Common Properties
+
+maintainers:
+  - Rob Herring <robh@kernel.org>
+
+# always select the core schema
+select: true
+
+properties:
+  thermal-zones:
+    anyOf:
+      - type: object # for nodes named 'thermal-zones'
```

Comment:
We inherited this in other cases, but not great to introduce a new one. However, I don't have a better naming suggestion, so I guess it is fine.